fastapi/routing.py

```diff
@@ -116,25 +117,28 @@ async def serialize_response(
     is_coroutine: bool = True,
 ) -> Any:
     if field:
```

Comment:
```py
if not field:
    return jsonable_encoder(response_content)
```
To decrease the indentation level. The pattern is called early return.

Reply:
good point